docs/regression-tuna.md

```diff
@@ -47,10 +47,21 @@ Here's how to run the end-to-end experiments:
 
 Please see doc [here](experiments-jdiq2018.md)
 
+### [TREC2018](experiments-trec2018.md)
+
 ## Log
 
 The following log details whenever regression tests have changed in terms of effectiveness:
 
+### August 10, 2018
+
++ [commit ](https://github.com/castorini/Anserini/commit/)
+
+This commit adds the effectiveness verification testing for the TREC2018 CENTRE Task2:
+
++ [JDIQ2018 Effectiveness](experiments-jdiq2018.md)
```

Comment:
JDIQ mentioned below already.
We won't have effectiveness results for a few months... do we need this now?